components/Layout/SectionWrapper.jsx

```diff
@@ -19,7 +23,18 @@ const SectionWrapper = ({
   }, [backgroundType]);
 
   return (
-    <section className={`${customClasses} ${backgroundClasses} scroll-mt-[100px]`} id={id} style={backgroundImage ? { background: `url(${backgroundImage}) center`, backgroundSize: 'cover' } : undefined}>
+    <section
+      className={`${customClasses} ${backgroundClasses} scroll-mt-[100px]`}
+      id={id}
+      style={
+        backgroundImage
+          ? {
+            background: `url(${backgroundImage}) center`,
+            backgroundSize: 'cover',
+          }
+          : undefined
```

Comment:
I think better to use undefined here (like we didn't pass any properties at all), wdyt
it's not recommended to use inline styles in react components, I'd like to avoid passing empty object